platforms/java/versionsToBuild.txt

```diff
@@ -6,4 +6,5 @@
 12.0.2, 10, 1202f536984c28d68681d51207a84b6c76e5998579132d3fe1b8085aa6a5f21e
 13.0.2, 8, 9ccc063569f19899fd08e41466f8c4cd4e05058abdb5178fa374cb365dcf5998
 14.0.2, 12, 7d5ee7e06909b8a99c0d029f512f67b092597aa5b0e78c109bd59405bbfa74fe
-17.0.1, 12, 1c0a73cbb863aad579b967316bf17673b8f98a9bb938602a140ba2e5c38f880a
\ No newline at end of file
+17.0.1, 12, 1c0a73cbb863aad579b967316bf17673b8f98a9bb938602a140ba2e5c38f880a
```

Comment:
For updating java versions, are we missing to update the following files ? -
1. build/__javaVersions.sh
2. doc/runtimes/java.md
3. platforms/java/getJavaSdk.sh
4. src/BuildScriptGenerator/Java/JavaVersions.cs

Reply:
We're supporting dynamic installation for these versions, the above files you mentioned we only update it for runtime base images. So here no need to update them. Same for other platforms.

Reply:
That's good to know. Thanks!